### PR TITLE
Serve schemas at /schema/akn-modified.xsd, caselaw.xsd and xml.xsd from Github.

### PR DESCRIPTION
**Pull request by dragon-dxw**
Questions to answer:

Is these the right URL?

> Also, the “schemaLocation” on legislation.gov.uk is http://www.legislation.gov.uk/schema/legislation.xsd, so maybe we could use https://caselaw.nationalarchives.gov.uk/schema/filename.xsd -- Jim

Is this the right approach?

Should /akn redirect here?

### CODE REVIEW

File: config/converters.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class SchemaFileConverter:
+    regex = r"(?:xml|caselaw|akn-modified)\.xsd"
```
Comment (pezholio) [inline config/converters.py:2]: Could we add `https://raw.githubusercontent.com/nationalarchives/ds-caselaw-marklogic/main/src/main/ml-schemas/` as a constant here? (Something like `SCHEMA_ROOT`) then return the full URL in `to_url`?

**Review by pezholio — COMMENTED** [review]
Just one comment about making the Schema Root a constant, but looks good I think.

I was thinking about potentially pulling down the schema on deploy, but actually caching is probably a more foolproof way of doing things.

File: config/converters.py (hunk 1 above)
Comment (dragon-dxw) [reply config/converters.py:2]: Rewritten the failing tests and added SCHEMA_ROOT. They failed because I turned caching on...

**Review by pezholio — APPROVED** [review]
LGTM 👍

**Pull request merged**